AdminWebsite/AdminWebsite.UnitTests/Services/HearingServiceTests.cs

```diff
@@ -43,23 +46,49 @@ public async Task should_send_confirmation_email_to_all_participants_except_a_ju
         public async Task should_not_send_confirmation_email_when_hearing_is_generic_case_type()
         {
             _hearing.Case_type_name = "Generic";
-            var judge = _hearing.Participants.First(x => x.User_role_name == "Judge");
             await _service.SendHearingConfirmationEmail(_hearing);
 
             _mocker.Mock<INotificationApiClient>()
                 .Verify(
-                    x => x.CreateNewNotificationAsync(It.Is<AddNotificationRequest>(r => r.ParticipantId != judge.Id)),
+                    x => x.CreateNewNotificationAsync(It.IsAny<AddNotificationRequest>()),
                     Times.Never);
         }
         
         [Test]
-        public async Task should_send_amendment_email_to_all_participants_except_a_judge()
+        public async Task should_send_amendment_email_to_all_participants_except_a_judge_if_no_email_exists()
+        {
+            _hearing.Other_information = JsonConvert.SerializeObject(new OtherInformationDetails {JudgeEmail = null});
+
+            var secondHearing = InitHearing();
+            secondHearing.Scheduled_date_time = _hearing.Scheduled_date_time.AddDays(1).AddHours(3).AddMinutes(20);
+            await _service.SendHearingUpdateEmail(_hearing, secondHearing);
+
+            _mocker.Mock<INotificationApiClient>()
+                .Verify(
+                    x => x.CreateNewNotificationAsync(It.IsAny<AddNotificationRequest>()),
+                    Times.Exactly(3));
```

Comment:
This test could break if someone changes the participant count. Maybe use the participant count - the judge